src/l500/l500-device.cpp

```diff
@@ -283,7 +283,7 @@ namespace librealsense
             [=]() {
                 auto z16rot = std::make_shared<rotation_transform>(RS2_FORMAT_Z16, RS2_STREAM_DEPTH, RS2_EXTENSION_DEPTH_FRAME);
                 auto y8rot = std::make_shared<rotation_transform>(RS2_FORMAT_Y8, RS2_STREAM_INFRARED, RS2_EXTENSION_VIDEO_FRAME);
-                auto sync = std::make_shared<syncer_process_unit>(); // is_zo_enabled_opt );
+                auto sync = std::make_shared<syncer_process_unit>(nullptr, false); // is_zo_enabled_opt );
```

Comment:
Remove the comment

Reply:
Add `// disable logging on this internal syncer`

Reply:
done